Tribler/community/multichain/community.py

```diff
@@ -369,19 +373,14 @@ def get_edges(self, nodes=None):
                 self.page_rank.add_edge(edge[1], edge[0])
         return edges
 
-    def get_page_rank(self, public_key):
+    def get_page_ranks(self):
         """
-        Return the page rank of a certain node.
+        If the page ranks are not yet cached in self.ranks, then fetch and store them.
 
-        :param public_key: the public key of the given node
-        :return: the page rank of the given node
```

Comment:
The method does not have a return statement, so a `:return:` comment is not needed I guess?